DigitalLearningSolutions.Data.Tests/DataServices/TutorialContentDataServiceTests/ObjectiveTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.DataServices.TutorialContentDataServiceTests
+{
+    using System.Linq;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    internal partial class TutorialContentDataServiceTests
+    {
+        [Test]
+        public void GetObjectivesBySectionId_returns_objectives_correctly()

```

Comment:
We should make sure we test the OriginalTutorialId = 0 case and the OriginalTutorialId > 0 case. I'm fine with running some sql at the start of the test if that makes it easier / clearer.

Reply:
Are the ObjectiveTests now testing both OriginalTutorialId cases?

Reply:
The SQL in the new version of this fine changes one of the OTIDs to be more than 0, to test that it is returned correctly.